tutorials/proto_rename/renaming-proto.md

```diff
@@ -0,0 +1,127 @@
+# Renaming proto fields
+
+Changing a field name will not affect protobuf encoding or 
+compatibility between applications that use proto definitions 
+which differ only by field names.
+The binary protobuf encoding is based on tag numbers, so that 
+is what you need to preserve.
+ 
+## Contents of this tutorial  
+- <walkthrough-editor-open-file 
+  	    filePath="startup-os/tutorials/proto_rename/person.proto" 
+  	    text="person.proto">
+      </walkthrough-editor-open-file> 
+It's a proto definition. 
+We will rename some fields in next steps and see what will happen.
+
+- <walkthrough-editor-open-file 
+  	    filePath="startup-os/tutorials/proto_rename/PersonTool.java" 
+  	    text="PersonTool.java">
+      </walkthrough-editor-open-file> 
+Contains main method and methods to write and read protobinary file.
+
+## First launch
+- Run the command to build targets:
+```bash
+bazel build //tutorials/proto_rename:all
+```
+- Look at `writePerson()` method of <walkthrough-editor-open-file 
+                                    	    filePath="startup-os/tutorials/proto_rename/PersonTool.java" 
+                                    	    text="PersonTool.java">
+                                        </walkthrough-editor-open-file> class. 
+This method is creating a new Person which will be saved to `person.pb` file.
+- Run the command to creates `person.pb` file:
```

Comment:
Run the command to create a `person.protobin` file.

Similarly below.

Reply:
Done.